css/styles.css

```diff
@@ -1,3 +1,59 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins');
```

Comment:
Instead of this import I'd put this code in between the `<head>` tags in your html file.

```
<link rel="preconnect" href="https://fonts.gstatic.com">
<link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@1,100&display=swap" rel="stylesheet">
```

Reason is I believe this import will delay this CSS file from loading until the font loads. By doing the method I mentioned this CSS file will not need to wait for the font to download.

Reply:
This solution doesn't seem to work for me. It won't load the poppins font.